.tekton/basic-component-pipeline.yaml

```
@@ -231,24 +212,25 @@ spec:
       value:
       - VERSIONS_SUFFIX=$(params.output-tag-suffix)
       - MAIN_IMAGE_TAG=$(tasks.determine-image-tag.results.image-tag)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+    - name: CACHI2_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
     runAfter:
     - prefetch-dependencies
```

Here it would change to:
```suggestion
    runAfter:
    - determine-image-tag
    - prefetch-dependencies
```